package.json

```diff
@@ -48,6 +48,7 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.29.0",
     "eslint-plugin-prettier": "^5.0.0",
+    "metaschema": "^2.1.5",
```

Comment:
```suggestion
```

Reply:
Can we include a type schema in the 'd.ts' file, something like this:
```
interface ValidationResult {
  valid: boolean;
  errors: string[];
}

export class Schema {
  check(value: any): ValidationResult;
}
```
or is there another approach?

Reply:
For testing purposes, we can create a mock Schema class that mimics the functionality of the 'check' function based on the test requirements.